google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseAdminClientImplTest.java

```diff
@@ -246,6 +250,24 @@ public void updateDatabaseDdlOpAlreadyExists() throws Exception {
     assertThat(op.getName()).isEqualTo(originalOpName);
   }
 
+  @Test
```

Comment:
Should we add a unit test to check for the scenario where we do not pass in any fieldsToUpdate?

Reply:
Yeah I think we can add a test but I feel an integration test would be more suitable than a unit test because we don't perform any validation on the field mask in the client libraries - that happens directly in the SpanFE.

Reply:
Makes sense, thanks!